collectors/apps.plugin/apps_plugin.c

```diff
@@ -3601,12 +3608,15 @@ static void send_collected_data_to_netdata(struct target *root, const char *type
     }
     send_END();
 
+    kernel_uint_t usedfdpercentage = (kernel_uint_t) ((currentmaxfds * 100) / sysconf(_SC_OPEN_MAX));
```

Comment:
One last thing. You can reduce the scope of this variable. Place it between 3618 and 3619. Or use `(kernel_uint_t) ((currentmaxfds * 100) / sysconf(_SC_OPEN_MAX))` directly inside `fprintf()`.

Reply:
Moved it inside the if condition!